docs/Modules/ecdsa.mdx

```diff
@@ -38,6 +38,49 @@ One of the most popular modules, it bridges the ease of traditional web logins w
 2. **dApp Interaction**: Simplifies the process of logging into dApps using Ethereum messages thanks to EIP-1271 support.
 3. **Ownership Management**: Mirrors traditional ownership systems, providing a familiar framework within a more advanced blockchain setting.
 
+## SDK Guide 📘
+
+### Initializing ECDSA Module 🌟
+
+There is no need to do anything in order to use ECDSAOwnershipValidationModule, this module is set by default on every smart account.
+
+### Usage
+
+| Key                                                                                                              | Description                                                                                                                           |
```

Comment:
are these both required param to create the instance?

Reply:
No they're not required. Signer is required but config is optional